atom/renderer/atom_render_view_observer.cc

```diff
@@ -29,6 +31,11 @@ namespace atom {
 
 namespace {
 
+// A hack here:
+// Copy from net/grit/net_resources.h of chromium repository
```

Comment:
We should copy this header in libchromiumcontent, this value can change in future.